core/src/main/java/hudson/WebAppMain.java

```diff
@@ -262,6 +264,25 @@ public void joinInit() throws InterruptedException {
     }
 
     /**
+     * Creates the plugin manager to use, if customized.
+     * @return The plugin manager to use or {@code null} to use the default one.
+     */
+    private PluginManager createPluginManager(ServletContext context, File rootDir) {
```

Comment:
Would be nice to add CheckForNull and to annotate Hudson/Jenkins constructors. NIT for this PR